src/Models/HydrostaticFreeSurfaceModels/compute_hydrostatic_free_surface_tendencies.jl

```diff
@@ -283,7 +283,7 @@ end
 ##### Tendency calculators for an explicit free surface
 #####
 
-""" Calculate the right-hand-side of the free surface displacement (``η``) equation. """
+""" Calculate the right-hand-side of the free surface displacement equation. (``η``) """
```

Comment:
The first one is right (`η` is the symbol we use for the free surface displacement).